api/clients/disperser_client_v2.go

```diff
@@ -75,13 +75,25 @@ func NewDisperserClientV2(config *DisperserClientV2Config, signer corev2.BlobReq
 	}
 
 	return &disperserClientV2{
-		config: config,
-		signer: signer,
-		prover: prover,
+		config:     config,
+		signer:     signer,
+		prover:     prover,
+		accountant: accountant,
 		// conn and client are initialized lazily
 	}, nil
 }
 
+// PopulateAccountant populates the accountant with the payment state from the disperser.
+// This function is required to be called before using the accountant. Perhaps rename to Start()?
```

Comment:
This function isn't required if a valid accountant is passed in the constructor, right?
If you want to require this method, we can remove accountant as an argument from the constructor.

Reply:
Oh yeah, will update the comment